configs/transferwise.json

```diff
@@ -5,12 +5,16 @@
   ],
   "stop_urls": [],
   "selectors": {
-    "lvl0": ".Content h1",
-    "lvl1": ".Content h2",
-    "lvl2": ".Content h3",
-    "lvl3": ".Content h4",
-    "lvl4": ".Content h5",
-    "lvl5": ".Content h6",
+    "lvl0": {
+      "selector": "a.Nav__Link",
```

Comment:
This will select all links, not only the active one

Reply:
fixed https://github.com/algolia/docsearch-configs/pull/5031/commits/735f5905f2eb3e79d4d3e767b354fd6fd399ec54